--- mirage/perf_analysis/gem5/src/mem/cache/tags/Tags.py	2025-02-14 07:49:06.000000000 +0000
+++ ../../src_randseed_patch/Tags.py	2025-08-19 01:43:08.797664070 +0000
@@ -163,3 +163,6 @@
 
     #Data-Replacement policy
     data_repl_policy = Param.String("random","Can be [random,reuse]")
+
+    #Random Seed for Global Evictions (true/false)
+    randseed_global_evictions = Param.Bool(Parent.randseed_global_evictions, "Enables Random Seed for RNG Used for Global Evictions")
