--- C/victim.cpp	2025-02-14 07:49:06.000000000 +0000
+++ victim_low1/victim.cpp	2025-08-11 21:26:27.322564234 +0000
@@ -69,12 +69,13 @@
     uint8_t data;
     BYTE bKey[16] = { 0xff, 0xee, 0xdd, 0xcc, 0xbb, 0xaa, 0x99, 0x88, 0x77, 0x66, 0x55, 0x44, 0x33, 0x22, 0x11, 0x00 }; 
 
-    BYTE bInput[16];
+    BYTE bInput[16] = {0xa9, 0x67, 0xbf, 0xe5, 0x99, 0x19, 0xce, 0x4e, 0x70, 0xa6, 0xeb, 0x8c, 0x5f, 0xe1, 0x0c, 0x26};
+    
     std::random_device rd;
     std::mt19937 gen(rd());
     std::uniform_int_distribution<> dist(0, 255);
-    for (int i = 0; i < 16; i++)
-	    bInput[i] = dist(gen);
+    //for (int i = 0; i < 16; i++)
+    // bInput[i] = dist(gen);
     BYTE bExpKey[176];
     KeyExpansion(bKey, bExpKey);
     ComputeTBoxes();   
